homeassistant/components/motion_blinds/__init__.py

```diff
@@ -113,9 +113,22 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     entry.async_on_unload(entry.add_update_listener(update_listener))
 
+    # check multicast interface
+    check_multicast_class = ConnectMotionGateway(hass, interface=multicast_interface)
+    working_interface = await check_multicast_class.async_check_interface(host, key)
+    if working_interface != multicast_interface:
+        data = {**entry.data, CONF_INTERFACE: working_interface}
+        hass.config_entries.async_update_entry(entry, data=data)
+        _LOGGER.info(
```

Comment:
```suggestion
        _LOGGER.debug(
```

This looks like this should happen with normal operation

Reply:
well I guess if nothing in the network changes, the used interface schould also not change.
But in any case it is not really a consirn to the end-user so debug level is fine.